search.go

```diff
@@ -54,6 +55,12 @@ func (c *SearchController) Show(ctx *app.ShowSearchContext) error {
 		return ctx.BadRequest(jerrors)
 	}
 
+	hostString := ctx.RequestData.Host
+	urlRegexString := fmt.Sprintf("(?P<domain>%s)(?P<path>/work-item/list/detail/)(?P<id>\\d*)", hostString)
+	search.RegisterAsKnownURL("work-item-list-details", urlRegexString)
+	urlRegexString = fmt.Sprintf("(?P<domain>%s)(?P<path>/work-item/board/detail/)(?P<id>\\d*)", hostString)
+	search.RegisterAsKnownURL("work-item-board-details", urlRegexString)
```

Comment:
Long term we might want to store these somewhere so we can build a list of known clients. Then you could paste links between the different UIs and it would just work.